src/main/java/com/automation_boss/app/App.java

```diff
@@ -39,5 +45,47 @@ public static void main(String args[]) throws IOException, InterruptedException
         r1.join(firstAttendee);
         r1.join(secondAttendee);
         r1.describeInventory(myFile);
+        System.out.println("room Chair comparison");
+        if (roomChair.equals(roomChair2))
```

Comment:
Simplify the code. `roomChair.equals(roomChair2)` will return `true` or `false`. So, you can use it in the message directly.